### PR TITLE
Php 8.0

### PR DESCRIPTION
**Pull request by lbacik**
Quick test regarding the issue: https://github.com/thephpleague/oauth2-facebook/issues/62

I will add a longer comment to the mentioned issue :)

### CODE REVIEW

**Comment by dakorpar** [comment]
I believe php 5.6 should be thrown out now plus it's not supported by this version of phpunit anyway. If someone is still on php5.6 it has older version, besides that version is not supported for long time already...

**Comment by akira28** [comment]
I would also drop 7.1 and 7.2 as the are EOL

**Comment by f3l1x** [comment]
Hi, can I ask how is it going? Do you need any help? Thank you.

**Comment by hemberger** [comment]
We probably want to change the tested php versions in `.travis.yml` as well. Do you plan to update this PR @lbacik?

@shadowhand Would you be willing to merge this once the tests pass? Are there any other issues that need to be resolved? I'm happy to help in any way to get a release going.

I'm a former user of https://github.com/facebookarchive/php-graph-sdk, which seems completely abandoned at this point (and, sadly, has the same primary author as this library), but I'm more hopeful that this library can be updated. Fingers crossed! :)

**Comment by sakonn** [comment]
I would be also thankful for this update.

**Comment by lbacik** [comment]
> We probably want to change the tested php versions in .travis.yml as well. Do you plan to update this PR @lbacik?

@hemberger - yep, of course - done :)

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,9 @@
 language: php
 
 php:
-  - 5.6
-  - 7.0
-  - 7.1
-  - 7.2
```
Comment (shadowhand) [inline .travis.yml:7]: It's questionable to me to remove 5.6 testing, when we still have 5.6 listed as a supported PHP version.

File: tests/src/Provider/FacebookTest.php [hunk 2]
```diff
@@ -28,7 +30,7 @@ class FacebookTest extends TestCase
      */
     protected $provider;
 
-    protected function setUp()
+    protected function setUp(): void
```
Comment (shadowhand) [inline tests/src/Provider/FacebookTest.php:33]: For instance, this is not compatible with PHP 5.6. I know that it won't impact runtime, but tests won't run on 5.6.

**Comment by shadowhand** [comment]
I've merged this, and I am going to release it as a hotfix, but I think we should do a major release that makes a hard requirement on PHP 7.4+.

**Comment by shadowhand** [comment]
Released in version 2.0.4.

**Pull request merged**